spec/serializers/document_serializer_spec.rb

```diff
@@ -8,6 +8,11 @@
   let(:document) { create :document }
   let(:result) { ActiveModelSerializers::Adapter.create(serializer).serializable_hash }
 
+  before do
+    # This is needed because of use of #service_url, since these tests do not run the controllers.
+    ActiveStorage::Current.host = 'http://example.com'
+  end
```

Comment:
Would be preferable to move this to one of the spec helper files?

Reply:
..or maybe `test.rb` 🤔

Reply:
maybe, but we need this only in this test.

Reply:
added on test.rb

Reply:
no, it does not work on test.rb, 

> ActiveStorage::Current.host makes use of CurrentAttributes, a type of attributes that are set/reset on every request and can be used everywhere in Rails